tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +76,18 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That the results is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert tai.inventory == [item_a, item_b, item_f]
+    assert jesse.inventory == [item_d, item_e, item_c] 
```

Comment:
Nice assertions for all the relevant data! How could we check the right values are in the right list if we could not guarantee the order of the lists? This feedback applies to all the tests below where a vendor's inventory is compared to a list with a specific ordering.